pkg/controlplane/controller/leaderelection/leaderelection_controller_test.go

```diff
@@ -765,16 +772,18 @@ func TestController(t *testing.T) {
 					case <-ctx.Done():
 						return
 					case <-ticker.C:
-						for _, lc := range tc.createAfterControllerStart {
```

Comment:
For the reviewer: this was wrong because we have tc.candidates now too. So better list and answer all pings.

Reply:
ok, thanks!